app/Entirety/devices/views/devices.py

```diff
@@ -75,7 +75,18 @@ def get_devices_data(self):
         if not pattern:
             pattern = pop_data_from_session(request=self.request, key="search-pattern")
             pattern = "" if not pattern else pattern
-        devices = get_devices(self.project)
+        device_list = get_devices(self.project)
```

Comment:
So the user will not have any way to delete the invalid devices from the Entirety only, right?